conf/messages

```diff
@@ -430,11 +430,11 @@ lbl.na = Not applicable
 lbl.day = Day
 lbl.month = Month
 lbl.year = Year
-lbl.date.example = For example, 31 03 2017.
+lbl.date.example = For example, 31 03 2017
```

Comment:
this was recently changed, requested in AMLS-4656, looks like we are changing this back and forth :/

Reply:
Changed, full stop is to be kept.